content/help/faq/security/k8s-health-checks.md

```diff
@@ -13,8 +13,7 @@ communicate with the service with http traffic.
 
 You can use a separate port for health check and enable mutual TLS only
 on the regular service port. Refer to [Health checking of Istio
-services](/docs/tasks/traffic-management/app-health-check/)
-for more information.
+services](/help/ops/setup/app-health-check/) for more information.
 
 Another workaround is to use a [liveness command](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/#define-a-liveness-command)
```

Comment:
Would be really good to put a line around why user should use the "another workaround"... In my opinion, it should not be recommended unless there is a reason.

Reply:
Good point. I think it's mostly about the risk of a new feature. We're not sure if the app prober take has no bug. I have https://github.com/istio/istio/issues/10357 to track turning on rewrite by default.